src/results/AnswerHandler.js

```diff
@@ -0,0 +1,49 @@
+const constants = require("../constants");
+
+module.exports = class AnswerHandler {
+  constructor(resultsRepository, formRepository) {
+    this.repo = resultsRepository;
+    this.formRepo = formRepository;
+  }
+
+  handle(answer) {
+    const formId = answer.formId;
+    const form = this.formRepo.get(formId);
+    const results = this.repo.get(formId);
```

Comment:
tutaj getujesz z bazy więc await (sorry, że wcześniej nie zauważyłem), a skoro await to funkcja powinna być async i trzeba na nią awaitować